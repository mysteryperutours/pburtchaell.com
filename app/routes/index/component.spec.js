import React from 'react';
import IndexRoute from './component';

describe(IndexRoute.name, () => {
  it('should render component', () => {
    const wrapper = shallow(
      <IndexRoute />
    );

    expect(wrapper).toMatchSnapshot();
  });
});
