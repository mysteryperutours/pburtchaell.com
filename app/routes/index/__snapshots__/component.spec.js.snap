exports[`IndexRoute should render component 1`] = `
<RouteContainer>
  <Row
    defaultColumn={true}
    elementType="div">
    <Text
      style={
        Object {
          "marginBottom": "3rem",
        }
      }
      type="h1">
      Patrick Burtchaell is a product design intern at Facebook, studying design at Loyola University New Orleans.
    </Text>
  </Row>
  <Row
    defaultColumn={true}
    elementType="div">
    <Column
      elementType="div"
      size={8}
      style={Object {}}>
      <Text
        style={
          Object {
            "marginBottom": "8rem",
          }
        }
        type="h2">
        Previously, he worked for Fixt, Fileee, Gigster, NewAperio, and others.
      </Text>
    </Column>
  </Row>
  <Row
    defaultColumn={true}
    elementType="div">
    <Text
      type="h3">
      Selected Works
    </Text>
  </Row>
  <ListView
    isPending={true}
    itemStyle={Object {}}
    items={null}
    layoutType="GRID"
    listViewItem={[Function]} />
</RouteContainer>
`;
